### ISSUE TITLE
NFLGame from a Nodejs Express App

### ISSUE THREAD

**Issue by keithfabean** [BurntSushi/nflgame#249]
Is there a way to use nflgame from a nodejs express app?  I'm not a Python dev and I'm new to Nodejs so I'm not sure how to cross languages like this.  I use to be a dev about 12 years ago and am really rusty.  I'm trying my hand at something new.
I've looked at nfldb and I know the data I want is there but the volume of data is just way too much for the level of Heroku/Postgres that I have set up, and I'm not at a state that warrants the cost to upgrade.  That and I am having problems getting the data load portion of the nfldb scripts to work. I'm also having problems getting the Python app set up locally to keep the data synced.

Any insight or help would be appreciated.

**Comment by BurntSushi** [comment]
No, sorry, I don't know if a way. You're best bet is to use nfldb and hit
the database with node.

**Comment by kingram6865** [comment]
You could use Node.js to read the json data being delivered by the api.
Or as suggested use nfldb with node.

**Comment by BurntSushi** [comment]
Reading the JSON isn't straightforward unfortunately.

**Comment by kingram6865** [comment]
Ah. I see....

**Comment by keithfabean** [comment]
Yeah.  I’ve seen the JSON. Not very friendly.

I have the DB loaded locally.  Next up I’ll work on getting the Python module installed and working.  My other issue is about volume of data.  I have a Heroku/Postgres instance that I want to use for a small private app I’m creating for a group of guys I have a fantasy football league with.  My Heroku/Postgres instance won’t allow me that volume of data.  Is there a way to limit the data to just a year or 2?  If I clear the DB of all the data prior to 2015 will the ‘update’ script just reload it when that runs?

**Comment by shoesCodeFor** [comment]
It is possible.  Google 'Node and Django' integration.  It will not be easy but it is possible.  Good Luck
